src/query/service/src/pipelines/processors/transforms/aggregator/aggregate_exchange_injector.rs

```diff
@@ -67,9 +67,9 @@ struct AggregateExchangeSorting<Method: HashMethodBounds, V: Send + Sync + 'stat
 impl<Method: HashMethodBounds, V: Send + Sync + 'static> ExchangeSorting
     for AggregateExchangeSorting<Method, V>
 {
-    fn block_number(&self, data_block: &DataBlock) -> Result<isize> {
```

Comment:
we can generate a sorted block_number by `max_partition_count` and `bucket` number.

eg:

```
block_number = max_partition_count * 10000 + bucket;
```

Reply:
Thus we don't need to change the aggregate meta.